<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SniffDialog</class>
 <widget class="QDialog" name="SniffDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="edit_filter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>621</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push_start">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_stop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
